content/en/tracing/serverless_functions/enable_aws_xray.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Enable AWS X-Ray Tracing
+kind: documentation
+description: 'Trace your Lambda functions with AWS X-Ray'
+---
+## Enable AWS X-Ray
+
+As a first step, make sure to [install the AWS integration][1] before proceeding.
+
+Ensure the following permissions are present in the policy document for your AWS/Datadog Role:
+
+```text
+xray:BatchGetTraces,
+xray:GetTraceSummaries
+```
+
+The `GetTraceSummaries` permission is used to get the list of recent traces. `BatchGetTraces` actually returns the full traces themselves.
+
+Then, [enable the X-Ray integration within Datadog][2].
+
+If you are using a Customer Master Key to encrypt traces, add the `kms:Decrypt` method to your policy where the Resource is the Customer Master Key used for X-Ray.
+
+**Note:** Enabling the AWS X-Ray integration increases the amount of consumed Analyzed Spans which can impact your bill.
+
+### Enabling AWS X-Ray for your functions
+
+To get the most out of the AWS X-Ray integration, you'll need to enable it _on_ your Lambda functions and API Gateways, **and** install the tracing libraries _in_ your Lambda functions.
+
+#### Recommended: Using the Serverless Framework plugin
+
+The [Datadog Serverless Framework plugin][3] automatically enables X-Ray for your Lambda functions and API Gateway instances. The plugin also automatically adds the [Datadog Lambda Layer][4] to all of your Node and Python functions.
+
+[Get started with the Serverless Framework plugin][5] and [read the docs][3].
+
+Lastly, [install and import the X-Ray client library in your Lambda function](#installing-the-x-ray-client-libraries).
+
+#### Manual setup
+
```

Comment:
Why are we telling them this method, if it's not the recommended path? Under what conditions would they use it? (mention those here)